src/welldevdemomodel/io/SqlConnection.java

```diff
@@ -0,0 +1,122 @@
+package welldevdemomodel.io;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class SqlConnection {
+	private static Connection myconnection= null;
+	private static SqlConnection conn=null;
+	private static PreparedStatement ps= null;
+	private static ResultSet res= null;
+	private SqlConnection() {
+		
+	}
+	
+	public static SqlConnection connect() {
+		
+		if(conn==null)
+		{
+			conn=new SqlConnection();
+			getConnection();
+		}
+		
+		return conn;
+	}
+	public static Connection getConnection() {
+		
+		try {
+			if (myconnection==null) {
+			Class.forName("com.mysql.jdbc.Driver");
+			
+			try {
+			myconnection = DriverManager.getConnection("jdbc:mysql://localhost:3306/demo", "root", "root");
```

Comment:
"jdbc:mysql://localhost:3306/demo", "root", "root"
host, dtabase username, password in constants.. or make a configuration class